syft/pkg/cataloger/java/native_image_cataloger.go

```diff
@@ -0,0 +1,309 @@
+/*
+ * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
```

Comment:
I don't think we'd be able to accept a PR with a conflicting copyright

Reply:
We want this change to be in line with the project's license. Is it alright if we keep the following copyright header in this file to acknowledge the change comes from Oracle?

Copyright © 2022 Oracle and/or its affiliates.

Reply:
Thanks @wdblair, I've asked for some more clarity on this and will get back to you with a definitive answer.

But, I'm curious if there might be some alternate solution where you / Oracle can get credit for this contribution without starting to add copyright headers in the codebase (as you can see, we are not doing this currently). What if we added an `AUTHORS.txt` or `CONTRIBUTORS.txt`, which we could be explicit about people / organizations who have contributed to this codebase? Would that be an acceptable method to retain credit for contributing things?

Reply:
Thanks for following up on this, I see that the copyright headers would deviate from your project's style conventions. I'll check whether a designated file such as `AUTHORS.txt` or `CONTRIBUTORS.txt` file will be alright and update you here. Would it be alright if we include the copyright statement only within this designated file?

Reply:
Hi @kzantow, we have updated this PR to include a `CONTRIBUTORS.txt` file to document the change, and have removed the copyright header from the source file. Would this be alright, or should we change the format of the contributors file or place it somewhere else in the repo?

Reply:
Thank you @wdblair -- I asked about this `CONTRIBUTORS.txt`, and I think this is a lot closer to what we could take. The response I have for this is:

> We are okay with a notice of attribution along the following lines:
>
> _GraalVM Native Image - A cataloger contributed by Oracle, Inc. that extracts packages given within GraalVM Native Image SBOMs._
>
> But not okay with including in the attribution notice any copyright notice of Oracle or any other such external entity, because we have no way of knowing, other than taking the word of the contributing entity, whether and what extent, that entity is actually owner of copyright in the contribution.

Sorry this is challenging to get to the finish line -- it's the first time we've had a request like this in our open source projects.

Reply:
No worries, thanks for sharing the proposed change and for helping improve the PR. I will check to see if an attribution in this form is alright and will update you here.

Reply:
Hi @kzantow, this attribution works well for us. I have updated the `CONTRIBUTORS.txt` to follow this format.